doc/python/sphinx/grpc_asyncio.rst

```diff
@@ -0,0 +1,120 @@
+gRPC AsyncIO API
+================
+
+.. module:: grpc.experimental.aio
+
+Overview
+--------
+
+gRPC AsyncIO API is the **new version** of gRPC Python whose architecture is
+tailored to AsyncIO. Underlying, it is using C-Core's callback API, and
```

Comment:
This should be written for an audience who is not familiar with the internals of gRPC. If we use jargon ("Core", "callback API"), we either need to explain what it means in-line or link to something that *does* explain what it means.